src/index.js

```diff
@@ -40,19 +42,26 @@ function toMatchImageSnapshot(received, { customSnapshotIdentifier = '', customD
     customDiffConfig,
   });
   let pass = true;
```

Comment:
No need to set `pass` here you can simply set it as a `const` in line 56

Reply:
Doing that will mean it's not in scope for the return